pages/src/source/workers/enchanter.md

```diff
@@ -4,9 +4,23 @@ worker: enchanter
 layout: worker
 ---
 {% infobox_worker %}
-The Enchanter is a specialist part of your colony. The Enchanter will steal XP from other workers to create enchanted books that can be used on tools, weapons, and armor for your workers (or yourself). (The higher their Mana level, the more XP they'll collect per trip.) To do this, the Enchanter will also need an [ancient tome](../../source/items/ancient_tome), obtained from raiders. The ancient tome will be used up during this process.
 
-The higher the level of the Enchanter's Tower, the higher the level of the enchanted books the Enchanter will produce (the Enchanter's Knowledge level also contributes to this). So:
+The enchanter specializes in crafting enchanted books and magical scrolls with special abilities useful to the administration of colonies. 
+
+## Crafting Enchanted Books
+
+In order to create enchanted books, the enchanter can be configured to observe another worker to generate XP. The enchanter will consume the generated XP and an [ancient tome](../../source/items/ancient_tome) to create an enchanted book.  The ancient tome is dropped by raiders.
+
+### Stats
```

Comment:
Remove this table please, we don't include this on any page and I'm currently working on better integrating this information into the worker pages.